docs/shared/reference/sitemap.md

```diff
@@ -496,6 +496,12 @@
       - [noop](/packages/nx/executors/noop)
       - [run-commands](/packages/nx/executors/run-commands)
       - [run-script](/packages/nx/executors/run-script)
+  - [playwright](/packages/playwright)
```

Comment:
This could be a follow up PR, but we should also make sure to have an 'overview' page like all the other plugins, that walks through the basics of the plugin. I'll add the task to asana.